docroot/themes/humsci/humsci_basic/src/scss/partials/components/_card.scss

```diff
@@ -144,23 +144,30 @@
       @include visually-hidden;
     }
 
-    .field-content {
-      display: block;
-      padding: 0 hb-calculate-rems(4px) 0 hb-calculate-rems(28px);
-      line-height: 127%; // keep icons from being cropped at smaller screen sizes
+    display: block;
+    padding: 0 hb-calculate-rems(4px) 0 hb-calculate-rems(28px);
+    line-height: 127%; // keep icons from being cropped at smaller screen sizes
 
-      @include hb-themes(('colorful', 'airy')) {
-        font-size: hb-calculate-rems(16px);
-      }
+    &.hb-card__subcontent-detail {
```

Comment:
Is this necessary? You're already adding a right padding to all these divs in line 148.